pystemd/utils.pyi

```diff
@@ -1,20 +0,0 @@
-#
```

Comment:
Did you delete this one on purpose?

Reply:
yes, I moved the typing directly into the file `pystemd/utils.py`